framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -432,6 +450,29 @@ protected void addMainToArgs(final List<String> args) {
             args.add(quote(concatenatePaths(ppOpts)));
         }
 
+        if (PluginUtil.getJreVersion() > 8) {
+            // Get Patch_Module_argfile
+            final File patchModuleArgsFile = new File(jdkJar, "Patch_Modules_argfile");
+
+            // Get ${CHECKERFRAMEWORK}/checker/dist path
```

Comment:
Shouldn't this be relative to `jdkJar` as well? It seems odd that the `JDK_PATH_OPT` flag would only impact the location of the `"Patch_Modules_argfile"` file, but not where the actual modules are.

Reply:
I have changed this to be relative to jdkJar.